src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor({ health = 100, name }) {
```

Comment:
The constructor of the `Animal` class should not expect an object as a parameter, but rather separate parameters for `name` and `health`. The `health` parameter should have a default value of 100.